src/styles/mixins/_mediaqueries.css

```diff
@@ -0,0 +1,3 @@
+@custom-media --tablet (min-width: 720px);
+@custom-media --desktop (min-width: 960px);
```

Comment:
tbh, 960px desktop is a bit confusing, but understand this is a weird ratio

Reply:
It was 1120px originally, but there was no need to have it that high.

Figure, if we need to target other sizes, we would adjust naming etc accordingly.